nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
Yeah, I can do the models.

so :
-AppRelease.version becomes semantic_version.django_fields.VersionField 
- php_min_version and php_max_version becomes php_version_spec = semantic_version.django_fields.VersionField 
- dito with the other (min|max)_version fields i.e. platform, database and php extension.